### ISSUE TITLE
TypeError children is not a function

### ISSUE THREAD

**Issue by visibleajay** [soroushchehresa/axios-react#3]
Receiving children is not a function on using the library with Typescript `~3.7.2`. 

Also any hints on defining types for function params as `ts-ignore` is used in src code for these params.

<a href="https://ibb.co/KGs4ngv"><img src="https://i.ibb.co/Ct9CLyG/ts-undefined.png" alt="ts-undefined" border="0"></a><br /><a target='_blank' href='https://freeonlinedice.com/'></a><br />

**Comment by visibleajay** [comment]
Applied a `;` after `React Closing Braces` threw this error.

Suggest to use Props for Children using 
```
interface ChildrenProps {
    loading: boolean, 
    error: AxiosError<any> | null, 
    response: AxiosResponse<any> | null, 
    refetch: Function, 
    networkStatus: string | null;
}

interface Props {
  children(props: ChildrenProps): ReactChild | ReactChildren;
  skip: boolean,
  config: AxiosRequestConfig,
}
```

Thanks!.

**Comment by soroushchehresa** [comment]
@visibleajay Thank you for this report.
This issue fixed on [v2.0.7](https://github.com/soroushchehresa/axios-react/releases/tag/v2.0.7).
